test/modules/aws-sqs-integration.ts

```diff
@@ -0,0 +1,215 @@
+import * as iasql from '../../src/services/iasql';
+import {
+  defaultRegion,
+  execComposeDown,
+  execComposeUp,
+  finish,
+  getPrefix,
+  itDocs,
+  runBegin,
+  runCommit,
+  runInstall,
+  runQuery,
+} from '../helpers';
+
+const prefix = getPrefix();
+const dbAlias = 'sqstest';
+
+const begin = runBegin.bind(null, dbAlias);
+const commit = runCommit.bind(null, dbAlias);
+const query = runQuery.bind(null, dbAlias);
+const install = runInstall.bind(null, dbAlias);
+const region = defaultRegion();
+
+const modules = ['aws_sqs'];
+
+jest.setTimeout(360000);
+beforeAll(async () => await execComposeUp());
+afterAll(async () => await execComposeDown());
+
+let username: string, password: string;
+describe('AwsSQS Integration Testing', () => {
+  it('creates a new test db', done => {
+    (async () => {
+      try {
+        const { user, password: pgPassword } = await iasql.connect(dbAlias, 'not-needed', 'not-needed');
+        username = user;
+        password = pgPassword;
+        if (!username || !password) throw new Error('Did not fetch pg credentials');
+        done();
+      } catch (e) {
+        done(e);
+      }
+    })();
+  });
+
+  it('installs the aws_account module', install(['aws_account']));
+
+  it(
+    'inserts aws credentials',
+    query(
+      `
+          INSERT INTO aws_credentials (access_key_id, secret_access_key)
+          VALUES ('${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
+      `,
+      undefined,
+      false,
+      () => ({ username, password }),
+    ),
+  );
+
+  it('starts a transaction', begin());
+
+  it('syncs the regions', commit());
+
+  it(
+    'sets the default region',
+    query(
+      `
+          UPDATE aws_regions
+          SET is_default = TRUE
+          WHERE region = '${region}';
+      `,
+      undefined,
+      true,
+      () => ({ username, password }),
+    ),
+  );
+
+  itDocs('installs the SQS module', install(modules));
```

Comment:
it misses more itDocs here, to actually output code examples into documentation

Reply:
Added. Thanks for pointing out.